config/fpga/clk_delay.yml

```diff
@@ -1,3 +1,4 @@
 dt: 0.1e-6
 n_bits: 9
 t_per: 250.0e-12
+depth: 3
```

Comment:
In the low level emulation model, we can now handle PI delays that go from 0-1.5 periods of the 4 GHz clock (3 edges, not 2 as before)